bin/manage

```diff
@@ -616,9 +616,15 @@ async def bootstrap(cluster):
         # At this point, the cluster is self-managed
         # Bootstrap the sealed secrets on the CAPI cluster
         await bootstrap_sealed_secrets(ek_capi, path, kubeconfig_capi)
+        # Ask user if they want to use custom commit message
+        use_custom_message = click.prompt("Do you want to provide a custom message?", default=False)
```

Comment:
What exactly is the user expected to type here to confirm / reject? If the default is false then just pressing 'enter' will use the default message but, for example, if the user types 'no' then (I think) since any string in Python with length > 0 evaluates to boolean `True` that will give unexpected behaviour.

We probably need this to be something like
```suggestion
        use_custom_message = click.prompt("Do you want to provide a custom message? (y/n)", default="n")
        while use_custom_message not in ["y", "n"]:
            click.prompt("Please enter 'y' for yes or 'n' for no")
        if use_custom_message == "y":
            commit_message = click.prompt("Enter your custom commit message")
        else:
            commit_message = f"Bootstrap cluster - {cluster}"
```

Reply:
Maybe we can use [click.confirm](https://click.palletsprojects.com/en/stable/api/#click.confirm) instead but it also isn't clear to me from the documentation whether this would suffer from the same problem when typing 'no' so you would need to test that.

Reply:
Thanks @sd109, using click.confirm adds the [y/N] prompt for the user to choose but fails when the user types "no". Instead, I have updated the code to include the [y/N] prompt and tested successfully with (y, N, no, n, NO, YES). It defaults to False and when a user types a gibberish character, it re-prompts the user. Kindly review.

Reply:
> when a user types a gibberish character, it re-prompts the user

Are you sure? I don't see any `while` loop or similar in the current code changes that would cause it to re-prompt.